pkg/kubelet/cm/cpumanager/fake_cpu_manager.go

```diff
@@ -32,36 +32,36 @@ type fakeManager struct {
 }
 
 func (m *fakeManager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesReady, podStatusProvider status.PodStatusProvider, containerRuntime runtimeService, initialContainers containermap.ContainerMap) error {
-	klog.Info("[fake cpumanager] Start()")
+	klog.Info("Start()")
 	return nil
 }
 
 func (m *fakeManager) Policy() Policy {
-	klog.Info("[fake cpumanager] Policy()")
+	klog.InfoS("Policy()")
 	return NewNonePolicy()
 }
 
 func (m *fakeManager) Allocate(pod *v1.Pod, container *v1.Container) error {
-	klog.Infof("[fake cpumanager] Allocate (pod: %s, container: %s", pod.Name, container.Name)
+	klog.InfoS("Allocate", "pod", klog.KObj(pod), "containerName", container.Name)
 	return nil
 }
 
 func (m *fakeManager) AddContainer(pod *v1.Pod, container *v1.Container, containerID string) {
-	klog.Infof("[fake cpumanager] AddContainer (pod: %s, container: %s, container id: %s)", pod.Name, container.Name, containerID)
+	klog.InfoS("AddContainer", "pod", klog.KObj(pod), "containerName", container.Name, "containerID", containerID)
 }
 
 func (m *fakeManager) RemoveContainer(containerID string) error {
-	klog.Infof("[fake cpumanager] RemoveContainer (container id: %s)", containerID)
+	klog.InfoS("RemoveContainer", "containerID", containerID)
 	return nil
 }
 
 func (m *fakeManager) GetTopologyHints(pod *v1.Pod, container *v1.Container) map[string][]topologymanager.TopologyHint {
-	klog.Infof("[fake cpumanager] Get Container Topology Hints")
+	klog.InfoS("Get Container Topology Hints")
```

Comment:
Nit
```suggestion
	klog.InfoS("Get container topology hints")
```

Not sure why this is capitalised

Reply:
@ehashman